components/movementsensor/movementsensor_test.go

```diff
@@ -263,6 +264,7 @@ var (
 	ang     = spatialmath.AngularVelocity{5.5, 1.2, 2.3}
 	orie    = &spatialmath.EulerAngles{5.6, 1.3, 2.4}
 	compass = 123.
+	la      = r3.Vector{1.1, 5.6, 0}
```

Comment:
Reasonable people might disagree, but I think this is too short a name for how broad a scope it has. I got confused in client_test.go about where `la` came from, and had expected it to be local to that function. How about `linAcc`?